docs/third-party-tools/openbrush.md

```diff
@@ -33,9 +33,27 @@ code base ([detailed description](https://github.com/727-Ventures/openbrush-cont
 
 ### Wrapper around Traits
 
-OpenBrush simplifies cross-contract calls, a contract that implements a particular
-Trait is not needed to call it. A wrapper enables calling methods of that Trait
-from some contract in the network (do a cross contract call).
+Traditionally, if you want to do a cross-contract calling in ink!,
+you need to import the other contract as a dependency in the calling contract's project. 
+
+OpenBrush simplifies the process with the wrapper around traits.
+If you know that the other contract implements a trait, 
+you only need that trait definition and the address of the other contract 
+in order to call a method from the deployed third-party contract.
+The wrapper is an attribute macro for traits that can be used to automatically generate 
+a callable `{Contract}Ref` implementation.
```

Comment:
Actually, the name is defined by the user because the syntax is:
```rust
// You can create wrapper in the place where you defined the trait
// Or if you import **everything** from the file where you define trait
#[openbrush::wrapper]
type Trait1Ref = dyn Trait1;
```